util/kubectl.go

```diff
@@ -0,0 +1,106 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/cmd/apply"
+	"k8s.io/kubectl/pkg/cmd/get"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"os"
+)
+
+func KubectlGet(resource string, resourceName string, namespace string) (stdout string, stderr string, err error) {
+	kubeConfigFlags := genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag()
+	kubeConfigFlags.Namespace = &namespace
+	matchVersionKubeConfigFlags := cmdutil.NewMatchVersionFlags(kubeConfigFlags)
+
+	out := &bytes.Buffer{}
+	errOut := &bytes.Buffer{}
+
+	f := cmdutil.NewFactory(matchVersionKubeConfigFlags)
+
+	ioStreams := genericclioptions.IOStreams{
+		In:     os.Stdin,
+		Out:    out,
+		ErrOut: errOut,
+	}
+	cmd := get.NewCmdGet("kubectl", f, ioStreams)
+	args := []string{resource, resourceName}
```

Comment:
@rushtehrani Right here is where you would use "kubectl get".
- resource = "pod/pods | namespace"
- resourceName = "podName | namespaceName"

Reply:
Wait, what do you mean by options?

Reply:
For example `-o` in the scenario we discussed earlier:
```
kubectl get svc istio-ingressgateway -n istio-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
```